ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/index.tsx

```diff
@@ -47,6 +47,9 @@ const MaskedImage = ({
         mediaType={primaryMimeType || undefined}
         fallbackMediaType={fallbackMimeType || undefined}
         sizes="(min-width: 1008px) 660px, 100vw"
+        preload={true}
+        width={800}
+        height={533}
```

Comment:
Just curious about these sizes - where did we get the numbers from? Are they needed i.e. could we get away with the preload=true only?

Reply:
that's the aspect ratio in `style`. It's just put there to stop layout shift